.github/workflows/cli-release.yaml

```diff
@@ -54,14 +54,16 @@ jobs:
           ref: ${{ github.event.inputs.source_git_ref_override }}
       - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
       - name: Build CLI
-        run: scripts/cli/build_cli_release.sh "Ubuntu-24.04" "${{inputs.release_version}}" "${{inputs.skip_checks}}"
+        run: scripts/cli/build_cli_release.sh "Ubuntu-24.04" "${{inputs.release_version}}" "${{inputs.skip_checks}}" "false"
       - name: Upload Binary
         uses: actions/upload-artifact@v4
         with:
           name: cli-builds-ubuntu-24.04
           path: aptos-cli-*.zip
 
   # Generic linux should be the older version, more likely compatible
+  # This actually builds for non-SIMD processors, and an older GLIBC
+  # TODO: Possibly find a way to support a baseline GLIBC rather than just whatever is on 22.04
```

Comment:
The previous Ubuntu LTS (2-4 years old) is what most people use as "baseline", so 22.04 is right on.